packages/service-library/src/servicelib/aiohttp/monitoring.py

```diff
@@ -112,7 +112,7 @@
 kPLATFORM_COLLECTOR = f"{__name__}.collector_platform"
 kGC_COLLECTOR = f"{__name__}.collector_gc"
 
-SIMCORE_USER_AGENT_HEADER: Final[str] = "X-SIMCORE_USER_AGENT"
+SIMCORE_USER_AGENT_HEADER: Final[str] = "X-Simcore-User-Agent"
```

Comment:
why not having some sort of ``constants.js`` in the front end with these constants and have a test that cross-check these constants??

This could be the first of many others!